README.md

```diff
@@ -32,14 +32,9 @@ e.g. List the programs provided by a container:
 
     $ docker run wsinpg/ub-18.04-irods-clients-4.2.11:latest \
```

Comment:
This image name looks wrong too - should be the same as the image actually created

Reply:
Should be fixed in new commit.